src/xdocs/property_types.xml

```diff
@@ -714,12 +714,12 @@
               The brace must be alone on the line. For example:
               <div class="wrapper">
                 <pre>
-    try {
-        ...
-    <b>}</b>
-    finally {
-        ...
-    <b>}</b>
+                  try {
+                      ...
+                  <b>}</b>
+                  finally {
+                      ...
+                  <b>}</b>
```

Comment:
Please recheck web site, this might make rendering a bit too much right indented

Reply:
this is what i'm talkin about. The xml fomatter wants `<b>}</b>` to be indented with 18 spaces.
From its point of view, the document looks like
```xml
              <div class="wrapper">
                <pre>
    <b>}</b>
    <b>}</b>
                </pre>
```
with some text inside. And it think that `<b>` tag should be moved to right.